xml/System.Text.Json/Utf8JsonWriter.xml

```diff
@@ -1004,12 +1008,8 @@ The property name is escaped before writing.
         <param name="utf8PropertyName">The UTF-8 encoded property name of the JSON object to be written.</param>
         <param name="value">The value to be written as a JSON number as part of the name/value pair.</param>
         <summary>Writes a property name specified as a read-only span of bytes and a <see cref="T:System.Decimal" /> value (as a JSON number) as part of a name/value pair of a JSON object.</summary>
-        <remarks>
-          <format><![CDATA[
```

Comment:
I am not sure, but I think we need to keep the format tag/CDATA. cc @mairaw, @carlossanlop

Reply:
If there's just plan text, \<remarks>text\</remarks> will build OK. Otherwise, it's best in a CDATA tag. That applies both this this occurrence and the ones below.

Reply:
I guess I could update some other remarks tag that aren't plain texts as well.

Reply:
Updated. I'm hoping I haven't missed one...

Reply:
For my edification, what is considered "plain text" in this context?